explore/github-data/LAST_MASTER_UPDATE.txt

```diff
@@ -1,5 +1,5 @@
-2018-11-06
+2018-11-13
```

Comment:
In the future, can we leave the data updates as separate pull requests? It makes the reviewing and merging easier. :)

Reply:
Whoops, yes, this one should have been done separately.

What do you suggest for other pull requests that are dependent on changes to the data? E.g. #118 , many of the new elements on that branch rely on additional pieces of data that were previously not included, thus requiring a complete update to function.